modules/add.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Kindly remove the eslint disable line, it is always better to fix all issues in the code.